From b594cd90bd053061ea8437bd9f219ee1ced956da Mon Sep 17 00:00:00 2001
From: Alexei Podtelezhnikov <apodtele@gmail.com>
Date: Tue, 19 Jan 2021 07:35:38 -0500
Subject: [PATCH] * srd/base/ftlcdfil.c (FT_Library_SetLcdGeometry): Fix return
 value.

---
 ChangeLog           | 4 ++++
 src/base/ftlcdfil.c | 2 +-
 2 files changed, 5 insertions(+), 1 deletion(-)

// diff --git a/ChangeLog b/ChangeLog
// index 6faa683a0..6a72a8918 100644
// --- a/ChangeLog
// +++ b/ChangeLog
// @@ -1,3 +1,7 @@
// +2021-01-19  Alexei Podtelezhnikov  <apodtele@gmail.com>
// +
// +	* srd/base/ftlcdfil.c (FT_Library_SetLcdGeometry): Fix return value.
// +
//  2021-01-15  Nikolaus Waxweiler  <madigens@gmail.com>
//  
//  	[afshaper] Fix hb_ot_tags_from_script deprecation warning.
diff --git a/src/base/ftlcdfil.c b/src/base/ftlcdfil.c
index 1a2e4d4ff..f3725a3d7 100644
--- a/src/base/ftlcdfil.c
+++ b/src/base/ftlcdfil.c
@@ -428,7 +428,7 @@
 
     ft_memcpy( library->lcd_geometry, sub, 3 * sizeof( FT_Vector ) );
 
-    return FT_THROW( Unimplemented_Feature );
+    return FT_Err_Ok;
   }
 
 #endif /* !FT_CONFIG_OPTION_SUBPIXEL_RENDERING */
-- 
2.26.3

